docs/products/kafka/kafka-connect/howto/mqtt-source-connector.rst

```diff
@@ -0,0 +1,124 @@
+
+::
+
+    {
+        "name": "CONNECTOR_NAME",
+        "connect.mqtt.hosts": "tcp://<HOST>:<PORT>",
+        "connect.mqtt.kcql": "KCQL_STATEMENT",
+        "connector.class": "com.datamountaineer.streamreactor.connect.mqtt.source.MqttSourceConnector",
+        "connect.mqtt.username": "USERNAME",
+        "connect.mqtt.password": "PASSWORD",
+        "key.converter": "org.apache.kafka.connect.json.JsonConverter",
+        "connect.mqtt.service.quality": "1",
+        "value.converter": "org.apache.kafka.connect.json.JsonConverter"
+    }
+
+The configuration file contains the following entries:
+
+* ``name``: the connector name, replace ``CONNECTOR_NAME`` with the name you want to use for the connector.
+* ``connect.mqtt.hosts``, ``connect.mqtt.kcql``, ``connect.mqtt.username`` and ``connect.mqtt.password``: source RabbitMQ/MQTT parameters collected in the :ref:`prerequisite <connect_mqtt_rbmq_source_prereq>` phase. 
+* ``key.converter`` and ``value.converter``: The data converter used for this example Jason converter is used.
+* ``connect.mqtt.service.quality``: Specifies the ``Mqtt`` quality of service.  
+    
+Check out the `dedicated documentation <https://docs.lenses.io/5.0/integrations/connectors/stream-reactor/sources/mqttsourceconnector/#options>`_ for the full list of parameters.
+
+Create a Kafka Connect connector with the Aiven Console
+'''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+To create the connector, access the `Aiven Console <https://console.aiven.io/>`_ and select the Aiven for Apache Kafka® or Aiven for Apache Kafka Connect® service where the connector needs to be defined, then:
+
+1. Click on the **Connectors** tab
+2. Clink on **Create New Connector**, the button is enabled only for services :doc:`with Kafka Connect enabled <enable-connect>`.
+3. Select the **Stream Reactor MQTT Source Connector**
+4. Under the *Common* tab, locate the **Connector configuration** text box and click on **Edit**
+5. Paste the connector configuration (stored in the ``mqtt_source.json`` file) in the form
+6. Click on **Apply**
+
+.. Note::
+
+    The Aiven Console parses the configuration file and fills the relevant UI fields. You can review the UI fields across the various tab and change them if necessary. The changes will be reflected in JSON format in the **Connector configuration** text box.
+
+7. After all the settings are correctly configured, click on **Create new connector**
+8. Verify the connector status under the **Connectors** tab
+9. Verify the presence of the data in the target Apache Kafka topic, the topic name is the one defined in the ``KCQL_STATEMENT``
+
+Example: Create a MQTT source connector from Aiven CLI
+------------------------------------------------------
+The Connectors can be created also using the dedicated :ref:`Aiven CLI command <avn_service_connector_create>`
```

Comment:
We can however add this sentence to the end of the Console example with a `Tip`

Reply:
Tip added